### PR TITLE
[No QA] Update Sage Intacct `Connect` + `Configure` docs with images

### PR DESCRIPTION
**Pull request by johncschuster**
### Fixed Issues

$ https://github.com/Expensify/App/issues/50430
PROPOSAL:

### Tests

- [ ] Verify that no errors appear in the JS console

### QA Steps

// TODO: These must be filled out, or the issue title must include "[No QA]."

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

### CODE REVIEW

File: docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md [hunk 1]
```diff
@@ -76,6 +80,8 @@ The Time & Expenses (T&E) module is often included in your Sage Intacct instance
        - **Expense Report:** EXP
        - **Employee:** EMP
        - **Duplicate Numbers:** Select “Do not allow creation”
+
```
[the remark this answers is not in the document]
Comment (neil-marcellini) [reply docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md:83]: I'm able to fix this by putting 8 spaces in front of the image. Here's rough proof that it fixes the markdown rendering.

-------

### Step 2: Enable the Time & Expenses Module (Only required if exporting reimbursable expenses as Expense Reports)
The Time & Expenses (T&E) module is often included in your Sage Intacct instance, but if it wasn't part of your initial Sage Intacct setup, you may need to enable it. **Enabling the T&E module is a paid subscription through Sage Intacct. For information on the costs of enabling this module, please contact your Sage Intacct account manager**. It's necessary for our integration and only takes a few minutes to configure.
1. In Sage Intacct, go to the **Company menu > Subscriptions > Time & Expenses** and toggle the switch to subscribe.
2. After enabling T&E, configure it as follows:
   - Ensure that **Expense types** is checked:
   - Under **Auto-numbering sequences** set the following:
       - **Expense Report:** EXP
       - **Employee:** EMP
       - **Duplicate Numbers:** Select “Do not allow creation”

        Screenshot 2024-11-19 at 2 57 26 PM

    - To create the EXP sequence, **click on the down arrow on the expense report line and select **Add**:
       - **Sequence ID:** EXP
       - **Print Title:** EXPENSE REPORT
       - **Starting Number:** 1
       - **Next Number:** 2
3. Select **Advanced Settings** and configure the following:
- **Fixed Number Length:** 4
- **Fixed Prefix:** EXP
4. Click **Save**
5. Under Expense Report approval settings, ensure that **Enable expense report approval** is unchecked
6. Click **Save** to confirm your configurations.

File: docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md [hunk 2]
```diff
@@ -126,8 +132,14 @@ To enable Customization Services go to **Company > Subscriptions > Customization
...
 1. Go to **expensify.com > Settings > Workspaces > New Workspace**.
+
+![Image of creating a new Workspace in Expensify]({{site.url}}/assets/images/SageConnectCreatingWorkspace.png){:width="100%"}
+
 2. Name the workspace something like "Sage Intacct Test Workspace."
 3. Go to **Connections > Sage Intacct > Connect to Sage Intacct**.
+
+![Image of selecting the Sage Intacct integration in Expensify]({{site.url}}/assets/images/SageConnectEnableSage.png){:width="100%"}
+
 4. Select **Download Package** (You only need to download the file; we'll upload it from your Downloads folder later).
```
Comment (neil-marcellini) [inline docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md:143]: When this renders the numbering is off, it restarts after each image. You could try fixing this by indenting the images 4 spaces, or by using html tags for an [ordered list](https://www.w3schools.com/html/html_lists_ordered.asp). Try the indentation first. Lmk if you need help.


Google Chrome 2024-11-19 15 13 47

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
Thanks for the work. There are some rendering issues we should sort out, then it will be ready.

**Comment by johncschuster** [comment]
Thanks, @neil-marcellini! I'll go through my urgent Concierge tasks this morning and see if I can work through your comments this afternoon.

**Comment by johncschuster** [comment]
Ok! I've just tried the `8 spaces` formatting suggestion! Is it possible to regenerate the preview link?

**Comment by neil-marcellini** [comment]
Cool. I kicked off the [workflow for that](https://github.com/Expensify/App/actions/runs/11975924999), it should post here again soon.

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
Ah, so I only meant that you should indent that specific image by 8 spaces, since the content around it, a bulleted list within a bulleted list, was also indented 8 spaces.

I _think_ images will display well in Jekyll/Markdown if they match the indentation of content around it. Since you indented some others more than necessary, that appears to be messing those up 🙈. Example below. 

Since this is actually a bit tricky to get right without testing, I recommend you download the app repo to your machine and then follow the [set up instructions](https://github.com/Expensify/App/blob/1af3f9dc3e6f1cd11e0724b3f6268cdc041b1d1a/help/README.md) to run the help site locally.

First steps are the clone the App repo  `git clone https://github.com/Expensify/App.git` and then `git checkout johncshuster-patch-4.`. You can ask chatGPT or me for help if you get stuck. Good luck 🍀

**Comment by johncschuster** [comment]
Oh darn it! I misunderstood and thought the indentations were required for _all_ of the images 🤦 

Ok, I'll go back through and clean the indents up. Thanks!

**Comment by johncschuster** [comment]
Ok! I think I've understood what I did wrong and what I needed to do to fix it.

For starters, it sounds like there was _one_ image that was broken, because it needed to be indented to match the indentation of the list it was included in. I misunderstood and indented _every_ image. I've now un-indented the images that should never have been indented in the first place.

Fingers crossed these are the last edits 🤞

**Comment by johncschuster** [comment]
@neil-marcellini can you re-run the workflow above so I can preview the changes?

**Comment by neil-marcellini** [comment]
Cool. Sure I kicked it off here https://github.com/Expensify/App/actions/runs/12126383519

**Comment by neil-marcellini** [comment]
https://b58214f4.helpdot.pages.dev/

File: docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md (hunk 2 above)
Comment (neil-marcellini) [reply docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md:143]: The numbering is still off. I'll play around with it locally and see if I can get it fixed.

**Review by neil-marcellini — CHANGES_REQUESTED** [review]
Thanks for the updates. It all looks good now except for the numbering in one of the sections. I'll work on it myself and see if I can get it working, and if not maybe I'll have it use bullet points instead.

**Review by neil-marcellini — APPROVED** [review]
Ok, I got the numbering fixed and it all looks good to me. I'll have to request another reviewer since I made a commit.

**Comment by neil-marcellini** [comment]
An updated preview will deploy here https://github.com/Expensify/App/actions/runs/12145891367

**Review by NikkiWines — APPROVED** [review]
:shipit:

**Comment by neil-marcellini** [comment]
Merged with failing checklists since this is a help site change only

**Pull request merged**